content/en/docs/ops/best-practices/image-signing-validation/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Image Signing And Validation
+description: Describes how to use image signatures to verify the provenance of Istio images.
+weight: 35
+aliases:
+    - /docs/ops/best-practices/image-signing-validation/
+keywords: [install]
+owner: istio/wg-environments-maintainers
+test: n/a
```

Comment:
Might be interesting to actually add a test for this page if we don't actively check that the commands work elsewhere.